index.html

```diff
@@ -1,19 +1,115 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
+
 <body>
+    <header class="header">
+        <img src="file:///C:/Users/New%20User/Documents/GitHub/HTML-CSS-Coursework-Week1/HTML-CSS-Coursework-Week2/favicon.ico" alt="logo">
+        <nav class="nav-bar">
+            <ul>
+                <a>meet karma</a>
+                <a>how its works</a>
+                <div id="red"><a>store</a></div>
+                <a>blog</a>
+                <a>help</a>
+                <a>login</a>
+
+            </ul>
+        </nav>
+    </header>
+    <main class="main">
+
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+        <div class="wifi">
+            <section class="karma">
+                <h1>interducing karma</h1>
+                <p>bring WIFI with you, Everywhere you go.</p>
+                <button class="learn">learn more</button>
+            </section>
+        </div>
 
+        <div class="h3">
+            <h1>Everyone need a little karma</h1>
+        </div>
+        <div class="little">
+            <section class="internet">
+                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
+                <path fill="#F05A29"
+                    d="M122 66.463h-1.106v-48.41c0-3.033-2.468-5.5-5.5-5.5H13.521a5.506 5.506 0 00-5.5 5.5v38.126h-2.02a5.506 5.506 0 00-5.5 5.5v51.996c0 3.032 2.467 5.5 5.5 5.5h35.546c3.033 0 5.5-2.468 5.5-5.5V90.627H57.69v10.04h-5.422a1.5 1.5 0 100 3H81.5a1.5 1.5 0 000-3h-5.421v-10.04h23.944v16.562c0 3.032 2.468 5.5 5.5 5.5H122c3.032 0 5.5-2.468 5.5-5.5V71.963c0-3.033-2.468-5.5-5.5-5.5zm-77.953 47.212c0 1.379-1.122 2.5-2.5 2.5H6.001a2.503 2.503 0 01-2.5-2.5v-4.286h40.546v4.286zm0-7.286H3.501v-44.71c0-1.378 1.122-2.5 2.5-2.5l35.546-.001c1.378 0 2.5 1.122 2.5 2.5v44.711zm29.032-5.722h-12.39v-10.04h12.39v10.04zm-26.032-13.04v-9.548h52.976v9.548H47.047zm52.976-15.664v3.116H47.047V61.678c0-3.033-2.467-5.5-5.5-5.5l-30.525.001V18.053c0-1.378 1.122-2.5 2.5-2.5h101.873c1.379 0 2.5 1.122 2.5 2.5v48.41h-12.371a5.507 5.507 0 00-5.501 5.5zm24.477 35.226c0 1.379-1.121 2.5-2.5 2.5h-16.477a2.502 2.502 0 01-2.5-2.5v-4.08H124.5v4.08zm0-7.08h-21.477V71.963c0-1.379 1.121-2.5 2.5-2.5H122c1.379 0 2.5 1.121 2.5 2.5v28.146z" />
+                </svg>
```

Comment:
```suggestion
                <img src="img/icon-devices.svg">
```
We can link to the `icon-devices.svg` image instead